shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -397,7 +397,7 @@ class WebFrameRenderer : public gin::Wrappable<WebFrameRenderer>,
 
  private:
   bool MaybeGetRenderFrame(v8::Isolate* isolate,
-                           const std::string& method_name,
+                           const char* method_name,
```

Comment:
This change seems unrelated, would you mind reverting it?

Reply:
done